Remove reference to __android_log_error_write which does not exist
on Android 5. See:

  https://github.com/termux/termux-packages/issues/1306

diff -u -r ../../pristine-src/libziparchive/zip_archive.cc ./zip_archive.cc
--- ../../pristine-src/libziparchive/zip_archive.cc	2017-08-23 21:25:26.492287975 +0200
+++ ./zip_archive.cc	2017-08-23 21:31:15.904280978 +0200
@@ -272,11 +272,6 @@
   if (static_cast<off64_t>(eocd->cd_start_offset) + eocd->cd_size > eocd_offset) {
     ALOGW("Zip: bad offsets (dir %" PRIu32 ", size %" PRIu32 ", eocd %" PRId64 ")",
         eocd->cd_start_offset, eocd->cd_size, static_cast<int64_t>(eocd_offset));
-#if defined(__ANDROID__)
-    if (eocd->cd_start_offset + eocd->cd_size <= eocd_offset) {
-      android_errorWriteLog(0x534e4554, "31251826");
-    }
-#endif
     return kInvalidOffset;
   }
   if (eocd->num_records == 0) {
@@ -388,9 +383,6 @@
   for (uint16_t i = 0; i < num_entries; i++) {
     if (ptr > cd_end - sizeof(CentralDirectoryRecord)) {
       ALOGW("Zip: ran off the end (at %" PRIu16 ")", i);
-#if defined(__ANDROID__)
-      android_errorWriteLog(0x534e4554, "36392138");
-#endif
       return -1;
     }
 
